aroquev/src/main/webapp/script.js

```diff
@@ -192,7 +192,21 @@ function getRandomCar() {
  * Fetch message from Java servlet and add it to the DOM
  */
 function getServletMessage() {
-  fetch('/data').then(response => response.text()).then((greeting)=> {
-    document.getElementById('js-servlet-container').innerHTML = greeting;
+  fetch('/data').then(response => response.json()).then((comments)=> {
+    // Get the cities container
+    const citiesContainer = document.getElementById('js-servlet-container');
+    citiesContainer.innerHTML = '';
+    
+    for (comment of comments) {
```

Comment:
`for (let comment of comments) {...}`

Otherwise it's setting `window.comment` each time through the loop.

Reply:
should I use const or let? Const is for when the comment remains unchanged right?

Reply:
Yes use `const` here. When to use `let` or `const` when either would work is a matter of preference. I think the general consensus is "use const wherever possible", and "choose one way and stay consistent".

Reply:
Done.